internal/models/helmcharts/fullname.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package helmchartorgmodel
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartFullName Full name of the helm chart.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.organization.fluxcd.helm.repository.chartmetadata.chart.FullName
+type VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartFullName struct {
```

Comment:
MetaData is expected?

Reply:
this is the fullname name which I've taken from swagger api model 
model is at this level `vmware.tanzu.manage.v1alpha1.organization.fluxcd.helm.repository.chartmetadata.chart.FullName`
so I think metadata is expected.